app/assets/javascripts/discourse/app/components/d-modal.hbs

```diff
@@ -33,9 +37,25 @@
         )
       }}
         <div class={{concat-class "d-modal__header" @headerClass}}>
-
           {{yield to="headerAboveTitle"}}
 
+          {{#if
+            (and
+              this.site.mobileView
+              this.dismissable
+              (has-block "headerPrimaryAction")
+            )
+          }}
+            <div class="d-modal__dismiss-action">
+              <DButton
+                @label="cancel"
```

Comment:
Please capitalise

Reply:
Not showing in screenshot, but I actually did it.